cmd/influxd/launcher/launcher.go

```diff
@@ -1078,7 +1092,43 @@ func (m *Launcher) run(ctx context.Context) (err error) {
 		}
 		transport = "https"
 
-		m.httpServer.TLSConfig = &tls.Config{}
+		// Sensible default
+		var tlsMinVersion uint16 = tls.VersionTLS12
```

Comment:
What was this the exact previous default?

Reply:
1.1 - we can use that as a default if preferred, but 1.2 has been around since 2008

Reply:
I'm happy moving default up to 1.2; perhaps this will need documenting somewhere

Reply:
I'd also opt for 1.2 as the safe minimum. If you can control it, and don't need to support old legacy clients, forbid to enable 1.1 or below entirely :)